### PR TITLE
Implement client-side performance enhancements

### PR DESCRIPTION
**Pull request by nrwinner**
This PR introduces several measures to improve client performance:

1. Removes custom implementation of ngx-dropzone-wrapper and replaces it with the original version
2. Update code to be compatible with ahead-of-time (AOT) compiling instead of just-in-time (JIT)
3. Compress images and standard assets to reduce load time
4. Lazy load the Learning Object Builder to both improve performance and remove the arrow function syntax since it's incompatible with AOT.

The CI build command needs to be updated yet to run builds with AOT

### CODE REVIEW

**Comment by DonSeannelly** [comment]
Can we update the CI config within this PR as well?

File: .circleci/config.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ jobs:
           command: npm test -- auth/
       - run:
           name: Build Application
-          command: yarn ng build --prod --aot=false --build-optimizer=false --no-progress
+          command: yarn ng build --prod --no-progress
```
Comment (gshaw1997) [inline .circleci/config.yml]: :tada: Woohoo!

File: src/app/cube/home/components/splash/splash.component.ts [hunk 2]
```diff
@@ -17,7 +17,7 @@ export class SplashComponent implements OnInit {
 
   ngOnInit() { }
 
-  submitSearch(event: KeyboardEvent) {
+  submitSearch(event: KeyboardEvent = undefined) {
```
Comment (gshaw1997) [inline src/app/cube/home/components/splash/splash.component.ts]: Comment please

Comment (nrwinner) [reply src/app/cube/home/components/splash/splash.component.ts]: No d2dedc8

**Pull request merged**